pages/linux/pkginfo.md

```diff
@@ -0,0 +1,19 @@
+# pkginfo
+
+> Query the package database on a CRUX system
+
+- list installed packages and their versions
+
+`pkginfo -i`
+
+- list files owned by a package
+
+`pkginfo -l {{package-name}}`
+
+- list the owner(s) of a file matching a pattern
```

Comment:
I'd change it to "files matching a pattern", since that's what actually happens.